common/io/Makefile.mk

```diff
@@ -47,7 +47,8 @@ test_programs += \
     common/io/MemoryBlockTester \
     common/io/SelectServerTester \
     common/io/StreamTester \
-    common/io/TimeoutManagerTester
+    common/io/TimeoutManagerTester \
+    common/io/SerialLockTester
```

Comment:
Please add this new test to .gitignore.

Reply:
It's actually already covered by these lines in .gitignore:
```
*Tester
*Tester.exe
```